spec/app/models/mdm/web_page_spec.rb

```diff
@@ -3,6 +3,48 @@
 
   context 'associations' do
     it { is_expected.to belong_to(:web_site).class_name('Mdm::WebSite') }
+
+    context 'serialized cookie attribute' do
+      let(:valid_cookie) do
+        {
+          name: 'test name',
+          value: 'test value'
+        }
+      end
+
+      let(:invalid_cookie) do
```

Comment:
This invalid cookie looks like encoded YAML.  Are you saying that if we have YAMLized cookies we're not de-YAMLing them to fix the data so it's usable as a Hash we're just passing it through? I get that fixes the bug of crashing the import, but that still means the data is unusable.  Doesn't it break code that expects a Hash and then gets String?